Source/WebCore/html/HTMLDialogElement.cpp

```diff
@@ -104,7 +105,8 @@ ExceptionOr<void> HTMLDialogElement::showModal()
     if (isPopoverShowing())
         return Exception { ExceptionCode::InvalidStateError, "Element is already an open popover."_s };
 
-    if (!protectedDocument()->isFullyActive())
+    Ref document = this->document();
```

Comment:
Please revert his change, this is a step in the wrong direction.

Reply:
Disregard this comment, I thought this variable was only used once on the next line. But it is used later in this function, a local variable is the right thing to do in this case. Reading diffs is hard sometimes :P